modules/dreamview/backend/handlers/websocket_handler.cc

```diff
@@ -129,7 +129,7 @@ bool WebSocketHandler::SendData(Connection *conn, const std::string &data,
   int ret = 0;
   PERF_BLOCK(absl::StrCat(name_, ": Writing ", data.size(),
                           " bytes via websocket took"),
-             0.1) {
+             0.02) {
```

Comment:
reducing the threshold to 0.02s will cause too frequent logs in production setting